gtsam/linear/LinearSolverParams.cpp

```diff
@@ -0,0 +1,5 @@
+//
```

Comment:
doxygen. Does this file need to exist?

Reply:
Will change if there is nothing to implement

Reply:
Also, does all the `Ordering` stuff actually belong to `LinearSolverParams`?

Reply:
Yeah, it’s an important property for direct solvers...